src/main/scala/com/raphtory/core/graph/visitor/Vertex.scala

```diff
@@ -1,11 +1,17 @@
 package com.raphtory.core.graph.visitor
 
+import PropertyMergeStrategy.PropertyMerge
+import VertexMergeStrategy.VertexMerge
+import com.raphtory.core.graph.visitor.EdgeDirection.Direction
+
 import scala.reflect.ClassTag
 
 trait Vertex extends EntityVisitor {
```

Comment:
Thank you for taking up this beast of a function overloading implentation 🥲 does the trick and I don't think there is a better way of doing it really, @miratepuffin ?